.circleci/config.yml

```diff
@@ -26,19 +26,11 @@ commands:
             php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
             EXPECTED_SIGNATURE=$(curl -s https://composer.github.io/installer.sig)
             ACTUAL_SIGNATURE=$(php -r "echo hash_file('sha384', 'composer-setup.php');")
-            [[ "$EXPECTED_SIGNATURE" == "$ACTUAL_SIGNATURE" ]] && sudo php composer-setup.php --install-dir=/bin --filename=composer || exit 1
-
-  conditionally-skip:
```

Comment:
Removed in https://github.com/studiopress/genesis-custom-blocks/pull/41/commits/5c8b60c170ea64a877e520be146300daa6ef87c2, as it's not practical to skip PHPUnit tests if this PR also includes PHP changes